src/views/dashboard/pages/UserProfile.vue

```diff
@@ -125,6 +140,16 @@
   import { mapMutations } from 'vuex'
   import { signin } from '@/api'
   export default {
+    props: {
+      submitted: {
+        type: Boolean,
+        default: false,
+      },
+      error: {
+        type: Boolean,
+        default: false,
+      },
+    },
```

Comment:
the way the alerts are shown will be more in the style of :

https://github.com/baobabsoluciones/cornflow-admin/blob/93b198d3b851595ee35cebf5606726ded0ce4695/src/views/dashboard/main/InstanceMenu.vue#L116-L119
and
https://github.com/baobabsoluciones/cornflow-admin/blob/93b198d3b851595ee35cebf5606726ded0ce4695/src/views/dashboard/main/InstanceMenu.vue#L136-L145